views/partials/password_tip_svg.hbs

```diff
@@ -12,7 +10,7 @@
       <g transform="translate(186.000000, 808.000000)">
         <g transform="translate(600.000000, 223.000000)">
           <g transform="translate(73.000000, 0.000000)">
-            <circle stroke="url(#linearGradient-1)" stroke-width="2" cx="42.5" cy="42.5" r="42.5"></circle>
+            <circle stroke="url(#passwordTipGradient-{{ @index }})" stroke-width="2" cx="42.5" cy="42.5" r="42.5"></circle>
```

Comment:
alternate proposal in #289 (although shouldn't block merging this PR, just a random thought).